opennms-base-assembly/src/main/filtered/etc/datacollection/ptpbase.xml

```diff
@@ -0,0 +1,218 @@
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.9.1.13" instance="ptpbaseClockPortRunningEntry" alias="ptpCloPorRunPacRece" type="counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.9.1.14" instance="ptpbaseClockPortRunningEntry" alias="ptpCloPorRunPacSent" type="counter64"/>
+   </group>
+   <group name="ptpbaseClockPortTransDSTable" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.1" instance="ptpbaseClockPortTransDSEntry" alias="ptCloPorTraDSDomInd" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.2" instance="ptpbaseClockPortTransDSEntry" alias="ptCloPorTraDSInsInd" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.3" instance="ptpbaseClockPortTransDSEntry" alias="ptClPoTrDSPorNumInd" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.4" instance="ptpbaseClockPortTransDSEntry" alias="ptCloPorTraDSPorIde" type="octetstring"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.5" instance="ptpbaseClockPortTransDSEntry" alias="ptClPoTrDSlMiPdReIn" type="integer32"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.6" instance="ptpbaseClockPortTransDSEntry" alias="ptCloPorTraDSFauFla" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.10.1.7" instance="ptpbaseClockPortTransDSEntry" alias="ptClPoTrDSPeMePaDel" type="octetstring"/>
+   </group>
+   <group name="ptpbaseClockPortAssociateTable" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.1" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorCurDomInde" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.2" instance="ptpbaseClockPortAssociateEntry" alias="ptClPorCurCloTypInd" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.3" instance="ptpbaseClockPortAssociateEntry" alias="ptClPorCurCloInsInd" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.4" instance="ptpbaseClockPortAssociateEntry" alias="ptClPorCurPorNumInd" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.5" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssPorInde" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.6" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssAddType" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.7" instance="ptpbaseClockPortAssociateEntry" alias="ptpClocPortAssoAddr" type="octetstring"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.8" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssPacSent" type="counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.9" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssPacRece" type="counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.10" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssoInErro" type="counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.2.11.1.11" instance="ptpbaseClockPortAssociateEntry" alias="ptpCloPorAssOutErro" type="counter64"/>
+   </group>
+
+   <systemDef name="ptpBase">
+      <sysoidMask>.1.3.6.1.4.1.</sysoidMask>
```

Comment:
I'm not sure if we want to try fetching these metrics from any MIB2-compatible SNMP agent on this planet. What about binding it to an `SNMP-PTP` service instead? It allows you to keep the systemDef global and gives users some control over where they want to try to fetch these metrics from. Maybe we can add an SNMP-PTP detector for it?

Reply:
Enabled that for Arista devices for now.